db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, on_delete=models.CASCADE, related_name="skill_set"
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, on_delete=models.CASCADE, related_name="player_set"
+    )
+    guild = models.ForeignKey(
+        Guild, null=True, on_delete=models.SET_NULL, related_name="guilds"
```

Comment:
The `related_name='guilds'` for the `guild` field in the `Player` model might be misleading. It suggests a reverse relation from `Guild` to `Player` as if a `Guild` has multiple `guilds`. Consider using a more descriptive name like `players` to indicate the reverse relation from `Guild` to `Player`.